pkg/filesystem/delete.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
This is just for deleting subvolumes, right? I'd suggest just combining this with subvolume.go